dart/lib/src/protocol/debug_meta.dart

```diff
@@ -7,10 +7,14 @@ class DebugMeta {
 
   final List<DebugImage> _images;
 
-  /// An immutable list of dynamic libraries loaded into the process (see below).
+  /// The list of debug images contains all dynamic libraries loaded
```

Comment:
worth adding the `immutable` comment back

Reply:
✅